pkgs/applications/networking/browsers/firefox-bin/update.nix

```diff
@@ -5,19 +5,23 @@
 , gnused
 , gnugrep
 , curl
+, gnupg
 , baseName ? "firefox"
-, basePath ? "pkgs/applications/networking/browsers/firefox-bin" 
+, basePath ? "pkgs/applications/networking/browsers/firefox-bin"
 , baseUrl ? "http://archive.mozilla.org/pub/firefox/releases/"
 }:
 
 let
   version = (builtins.parseDrvName name).version;
   isBeta = builtins.stringLength version + 1 == builtins.stringLength (builtins.replaceStrings ["b"] ["bb"] version);
 in writeScript "update-${baseName}-bin" ''
```

Comment:
could you please turn this ``writeScript`` which I created some time ago to normal ``mkDerivation`` so that nobody seems the horrible ``PATH`` trickery I had to do ;)